sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md

```diff
@@ -13,7 +16,7 @@ our efforts can be found in the [Azure SDK Design Guidelines for Java](https://a
 
 - Reactive streams support using [Project Reactor](https://projectreactor.io/).
 - Receive messages from all partitions of an Azure Event Hub using `EventProcessor`.
-- Provide an instance of `BlobPartitionManager` to your Event Processor. `BlobPartitionManager` uses Storage Blobs to 
+- Provide an instance of `BlobEventProcessorStore` to your Event Processor. `BlobEventProcessorStore` uses Azure Blob Storage to 
```

Comment:
I don't think we should be changing the old changelogs. Just mentioning it in the new one should be enough?

Reply:
It's the `Features` section that has the features description which can be updated. Changelogs section with preview and date shouldn't change.